File: onebot/src/main/java/cn/evole/onebot/sdk/event/UnsolvedEvent.java
package cn.evole.onebot.sdk.event;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * 未成功解析的事件
 *
 * @author MrXiaoM
 */
@Data
@EqualsAndHashCode(callSuper = true)
public class UnsolvedEvent extends Event {
    public String jsonString;
}

File: onebot/src/main/kotlin/sdk/event/UnsolvedEvent.kt
package cn.evolvefield.onebot.sdk.event

import cn.evolvefield.onebot.sdk.event.Event
import lombok.Data
import lombok.EqualsAndHashCode

/**
 * 未成功解析的事件
 *
 * @author MrXiaoM
 */
@Data
@EqualsAndHashCode(callSuper = true)
class UnsolvedEvent : Event() {
    var jsonString: String = "{}"
}
